pkg/agent/nodeportlocal/k8s/annotations.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const NPLAnnotationStr = "nodeportlocal.antrea.io"
+const NPLServiceAnnotation = "nodeportlocal.antrea.io/enabled"
```

Comment:
can you use a `const` block?

```go
const (
    NPLAnnotationStr = "nodeportlocal.antrea.io"
    NPLServiceAnnotation = "nodeportlocal.antrea.io/enabled"
)
```

I would also like the name to be more uniform for both constants. Maybe `NPLAnnotationKey` and `NPLEnabledAnnotationKey`, or similar names.

Reply:
Done